src/devices/namron.ts

```diff
@@ -942,6 +942,24 @@ const definitions: Definition[] = [
             await reporting.activePower(endpoint);
         },
     },
+    {
+        zigbeeModel: ['4512772', '4512773'],
+        model: '451277(2/3)',
```

Comment:
Could you not use `(2/3)` here but use a whitelabel for the other device instead? Like: 

https://github.com/Koenkk/zigbee-herdsman-converters/blob/d0aafe21dfadd399a98cc9176d2276d318cbf2a7/src/devices/namron.ts#L917

Reply:
Shure, hopefully this looks better now ?